src/plugins/data-card/data-card-tile.tsx

```diff
@@ -207,6 +214,13 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
               </div>
             }
           </div>
+        </div>
+
+        <div className="panel sort">
+          <SortSelect model={model} onSortAttrChange={(e) => setSort(e)} />
```

Comment:
```suggestion
          <SortSelect model={model} onSortAttrChange={setSort} />
```
In case you want to make things a tiny bit more concise.

Reply:
oh thanks